### ISSUE TITLE
Option for icon above inline title to be on same line as inline title

### ISSUE THREAD

**Issue by redactedscribe** [FlorianWoelki/obsidian-iconize#286]
**Is your feature request related to a problem? Please describe.**
#190 made it possible for icons to be displayed near the title within a markdown document, but the positioning is above the title.

**Describe the solution you'd like**
I think it would look nicer if the icon (an emoji in my case) would be inline with the inline title (which I have turned on).

Previously, I'd name a file `<emoji><space><title>` and the effect would be that what this FR is asking for.

This:
![Obsidian_2023-11-01_13-12-41](https://github.com/FlorianWoelki/obsidian-iconize/assets/25013746/ebe4f394-84a0-47e4-a847-e07219c8b6c7)

Over this:
![Obsidian_2023-11-01_13-13-10](https://github.com/FlorianWoelki/obsidian-iconize/assets/25013746/2e025447-2a96-4d2f-bcd9-752c7ea82bc0)

**Describe alternatives you've considered**
Some CSS hack possibly?

Thanks.

**Comment by redactedscribe** [comment]
This is possible using a CSS snippet:

```css
.iconize-title-icon, .inline-title {
  display: inline-block !important;
}
```

**Comment by oschrenk** [comment]
That would be great! I added the CSS snippet but realised it doesn't work in source mode as you said

**Comment by redactedscribe** [comment]
@oschrenk It's probably possible with better CSS. If you're not already heavily using Iconize, simply for emojis in titles and filenames etc, you can try [Front Matter Title](https://github.com/snezhig/obsidian-front-matter-title) and define your icon via `icon:` front matter. This is what I now use.

**Comment by suxscribe** [comment]
Another snippet option:

```
@media only screen and (min-width: 768px) {
  /* note title icon */
  .iconize-title-icon {
    position: absolute;
    line-height: 2.5em;
  }

  .inline-title {
    margin-left: 30px;
  }
}
```

`line-height` and `margin-left` may need some adjusting based on your theme and font-size.
Works in reading, editing and source mode

**Comment by FlorianWoelki** [comment]
This is now a native functionality in Iconize and has been available since the [v2.12.0](https://github.com/FlorianWoelki/obsidian-iconize/releases/tag/2.12.0).
